src/main/java/org/wmn4j/io/musicxml/MusicXmlWriterDom.java

```diff
@@ -392,6 +403,43 @@ private Element createClefElement(Clef clef) {
 		return clefElement;
 	}
 
+	private void handleMidMeasureClefChanges(
+			Element measureElement, Map<Duration, Clef> undealtClefChanges, Duration cumulatedDuration) {
+
+		List<Duration> offsets = new ArrayList<>(undealtClefChanges.keySet());
+		Collections.sort(offsets);
```

Comment:
No directly related to this pull request, but it would actually make a lot of sense to change the clef changes in Measure attributes to be handled as a `SortedMap` instead, so the offset values would be in ascending order.